site/pages/view-all-disruptions.page.tsx

```diff
@@ -15,6 +15,7 @@ const ViewAllDisruptions = ({
     filterStatus,
     enableLoadingSpinnerOnPageLoad = true,
     orgId,
+    showUnderground,
```

Comment:
```suggestion
    showUnderground = false,
```

We should default this value in case session is not returned.